src/StoreContext/StoreContext.tsx

```diff
@@ -490,17 +514,28 @@ export const StoreProvider = ({
       ]) => {
         const code = codeFromQuery || storedCode || exampleCode;
         const editorPosition = storedPosition;
-        const editorHeight = storedHeight;
-        const editorWidth = storedWidth;
+
+        const editorHeight =
+          (typeof storedHeight === 'string' ? storedHeight : null) ||
+          (typeof storedHeight === 'number' ? `${storedHeight}px` : null) ||
+          defaultEditorSize;
```

Comment:
These could be simplified further, if its a number convert to pixel, otherwise assume percentage, if falsey fallback to default.
```suggestion
          (typeof storedHeight === 'number' ? `${storedHeight}px` : storedHeight) ||
          defaultEditorSize;
```